### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by Sax593**

### CODE REVIEW

**Review by guenk — APPROVED** [review]
evil dead III

File: backend/src/controllers/usersHandlers.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      }
+    })
+    .catch((err) => {
+      console.error(err);
+      res.status(500).send("Error retriving data form database");
+    });
+};
+
+const getUserById = (req, res) => {
+  // const id = parseInt("req.params.id");
```
Comment (LoicBrassart) [inline backend/src/controllers/usersHandlers.js]: Dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One line of dead code... Too bad :p

**Review by guenk — APPROVED** [review]
10/10

**Review by LoicBrassart — APPROVED** [review]
No more Evils !

**Pull request merged**